src/tray.js

```diff
@@ -61,14 +63,38 @@ exports.create = function(config) {
         },
     ]);
 
+    blinkEnabled = config.blinkEnabled;
+
     const defaultIcon = nativeImage.createFromPath(config.icon_path);
 
     trayIcon = new Tray(defaultIcon);
     trayIcon.setToolTip(config.brand);
     trayIcon.setContextMenu(contextMenu);
     trayIcon.on('click', toggleWin);
 
+
     let lastFavicon = null;
+
+    let blinkInterval = null;
+    const currentIcon = {icon: defaultIcon};
+    const emptyIcon = nativeImage.createFromBitmap(Buffer.alloc(4, 0), {width: 1, height: 1});
```

Comment:
docs state this format is platform-dependent, so are we sure this works on all platforms & won't break in the future? What do we get from this - a single pixel with 0 alpha? Is `createEmpty()` not what we want? Is it not a problem that it's a different size to the normal icon?

Reply:
I tried `createEmpty()` first but it doesn't create an empty icon, it's like `null` instead of `0` so the icon doesn't disappear when I want it to. Looks like setting an empty icon created that way is a nop-like operation. I haven't noticed any issues with this on both Win10 and Linux, all is working as intended. I agree it's not the cleanest way possible but it's the least cumbersome one. I couldn't find a way of creating an empty image of desired size in Electron, only a bunch of methods for reading an existing image in predefined formats.